railties/lib/rails/engine/configuration.rb

```diff
@@ -85,6 +86,10 @@ def autoload_once_paths
       def autoload_paths
         @autoload_paths ||= paths.autoload_paths
       end
+
+      def webpacker_path
+        YAML.load_file("#{Rails.root}/config/webpacker.yml")[Rails.env]["source_path"]&.gsub("app/", "") if File.file?("#{Rails.root}/config/webpacker.yml")
```

Comment:
This code is too complex can you do a refactoring?

Also, we need to return `javascript` as a fallback in case the `webpacker` file doesn't exist.

Reply:
if i do this:
```
if File.file?("#{Rails.root}/config/webpacker.yml")
  YAML.load_file("#{Rails.root}/config/webpacker.yml")[Rails.env]["source_path"]&.gsub("app/", "")
else
  "javascript"
end
```

does that simplify enough?

Reply:
👍